Sources/MusicXML/Complex Types/Partwise/Partwise.Measure.swift

```diff
@@ -68,42 +74,40 @@ extension Partwise {
             optionalUniqueID: Int? = nil,
             musicData: [MusicData] = []
         ) {
-            self.number = number
-            self.text = text
-            self.implicit = implicit
-            self.nonControlling = nonControlling
-            self.width = width
-            self.optionalUniqueID = optionalUniqueID
+            self.attributes = MeasureAttributes(
+                number: number,
+                text: text,
+                implicit: implicit,
+                nonControlling: nonControlling,
+                width: width,
+                optionalUniqueID: optionalUniqueID
+            )
             self.musicData = musicData
         }
+
+        internal init(attributes: MeasureAttributes, musicData: [MusicData]) {
+            self.attributes = attributes
+            self.musicData = musicData
+        }
+
+        /// - Returns: A measure attribute.
+        public subscript <T> (dynamicMember keyPath: KeyPath<MeasureAttributes, T>) -> T {
+            return attributes[keyPath: keyPath]
+        }
```

Comment:
Aha! Gotcha

Reply:
I was always wondering when someone would want to use this…